src/ui/styles/Paragraph.scss

```diff
@@ -8,10 +8,10 @@
 }
 
 .paragraph--footer {
-  margin-left: -0.75rem;
   grid-area: footer;
   align-self: center;
   text-align: center;
+  margin: 0 1.5rem 1.5rem 1.5rem !important;
```

Comment:
Did setting the individual `margin-{direction}` values not work for this component? `!important` can become problematic

Reply:
It was getting overridden and would have required larger changes to the footer component to set otherwise.

Reply:
Lets go with it for now then